logs.txt

```diff
@@ -1,108 +1,3 @@
-2024-01-31T18:30:31Z: #13 CACHED
-2024-01-31T18:30:31Z: 
-2024-01-31T18:30:31Z: #12 [ 5/16] RUN /scripts/mark-builderror.sh 'cat /scripts/user_commands/env_vars.sh | bash'
-2024-01-31T18:30:31Z: #12 sha256:30e964357bd9f024f91d956bec94da689fa1b4baad39e12bef578dab8c1d166c
-2024-01-31T18:30:31Z: #12 CACHED
-2024-01-31T18:30:31Z: 
-2024-01-31T18:30:31Z: #10 [ 3/16] COPY --chown=yextci repo/package-lock.json /opt/buildhome/repo/package-lock.json
-2024-01-31T18:30:31Z: #10 sha256:09e63353ff946ecb140bd75007d42fd3af52efe68d89ea119fbc812a2749d163
-2024-01-31T18:30:31Z: #10 CACHED
-2024-01-31T18:30:31Z: 
-2024-01-31T18:30:31Z: #14 [ 7/16] WORKDIR /opt/buildhome/repo/.
-2024-01-31T18:30:31Z: #14 sha256:969a47afad22caa1c32e74de8cd4e437a353dbf7e1bf7e8ba5fd52e234ab75ef
-2024-01-31T18:30:31Z: #14 CACHED
-2024-01-31T18:30:31Z: 
-2024-01-31T18:30:31Z: #15 [ 8/16] COPY --chown=yextci user_commands/install_deps.sh /scripts/user_commands/install_deps.sh
-2024-01-31T18:30:31Z: #15 sha256:50082d2e65cd6d6a9a03801b5ab7833aa205c3eb29eb0fd0922bc26cba298c88
-2024-01-31T18:30:31Z: #15 CACHED
-2024-01-31T18:30:31Z: 
-2024-01-31T18:30:31Z: #16 [ 9/16] RUN --mount=type=ssh,id=deps_key,uid=2500 /scripts/mark-builderror.sh '/scripts/user_commands/install_deps.sh'
-2024-01-31T18:30:31Z: #16 sha256:28f3e8f100f5365902e8b041336774688bcdcd6968c84f0cf0e113fa2730dcaf
-2024-01-31T18:30:31Z: #16 CACHED
-2024-01-31T18:30:31Z: 
-2024-01-31T18:30:31Z: #17 [10/16] COPY --chown=yextci repo /opt/buildhome/repo
-2024-01-31T18:30:31Z: #17 sha256:b0ebbc1fe1f7f13612d7cfedb199b8c7029f15c06f18dbfab5ff7e63fd8010c8
-2024-01-31T18:30:31Z: #17 DONE 0.1s
-2024-01-31T18:30:31Z: 
-2024-01-31T18:30:31Z: #18 [11/16] COPY --chown=yextci user_commands/build_setup.sh /scripts/user_commands/build_setup.sh
-2024-01-31T18:30:31Z: #18 sha256:584fcb10e834da097740f69ef96b43574d5141c3cbc7fc351ae661003c59dfe6
-2024-01-31T18:30:31Z: #18 DONE 0.0s
-2024-01-31T18:30:31Z: 
-2024-01-31T18:30:31Z: #19 [12/16] RUN --mount=type=ssh,id=deps_key,uid=2500 /scripts/mark-builderror.sh '/scripts/user_commands/build_setup.sh'
-2024-01-31T18:30:31Z: #19 sha256:d0922746e08fef4ffa63b76f6e3c7cf8e1e9d10b66b8028c09546b2134c1816b
-2024-01-31T18:30:32Z: #19 DONE 0.9s
-2024-01-31T18:30:32Z: 
-2024-01-31T18:30:32Z: #20 [13/16] COPY --chown=yextci user_commands/build.sh /scripts/user_commands/build.sh
-2024-01-31T18:30:32Z: #20 sha256:205e7f57a910405148e3a9c0861419972cf38e97dcc30cb7ba49fe95efdb297c
-2024-01-31T18:30:32Z: #20 DONE 0.0s
-2024-01-31T18:30:32Z: 
-2024-01-31T18:30:32Z: #21 [14/16] RUN --mount=type=ssh,id=deps_key,uid=2500 /scripts/mark-builderror.sh '/scripts/user_commands/build.sh'
-2024-01-31T18:30:32Z: #21 sha256:eee710fb2ecb74bc61d382cb61ac9a2d47dafa5da407c315711b5382c90ffb11
-2024-01-31T18:30:33Z: #21 0.832 /scripts/user_commands/build.sh: line 2: [: too many arguments
-2024-01-31T18:30:34Z: #21 2.039 
 2024-01-31T18:30:34Z: #21 2.039 > pages-starter-blank@0.0.0 prebuild
```

Comment:
IMO the stuff that is really valuable to the PagesJS dev is from lines 106-201. This is the same as what they see as when they run `npm run prod` locally.